From 5af291015ba1be9ddaa32beb150f13049ae6ae07 Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Sat, 22 Sep 2018 14:14:54 +0200
Subject: [PATCH 17/18] overlay: Remove config_persistUsbDriveNotification

 * No one changes this overlay, so we assume it always be true

Change-Id: Iaa5674a7c5c20a2bad59acaea66629c720710a91
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index d2538737..9b135bb7 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -51,9 +51,6 @@
     <!-- Enable lockscreen rotation by default -->
     <bool name="config_enableLockScreenRotation">true</bool>
 
-    <!-- Whether to persist the notification for when a usb drive device is plugged in -->
-    <bool name="config_persistUsbDriveNotification">true</bool>
-
     <!-- Set this to true to enable the platform's auto-power-save modes like doze and
          app standby.  These are not enabled by default because they require a standard
          cloud-to-device messaging service for apps to interact correctly with the modes
-- 
2.17.1

